scripts/copy-tables.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import re
+
+from clickhouse_driver import Client
+
+SETTINGS = {
+    "load_balancing": "in_order",
+    "replication_alter_partitions_sync": 2,
+    "mutations_sync": 2,
+    "database_atomic_wait_for_drop_and_detach_synchronously": 1,
+}
```

Comment:
Where did you get these settings from? Could you elaborate why each of them are needed?

Reply:
i copied this over from `ClickhouseClientSettings.MIGRATE` but just doubled checked all of them and none actually apply to this script so removing